lib/stdlib/src/timer.erl

```diff
@@ -440,31 +444,14 @@ handle_call(_Req, _From, Tab) ->
               when Tab :: ets:tid().
 %% One-shot timer timeout.
 handle_info({timeout, TRef, {apply_once, MFA}}, Tab) ->
-    case ets:take(Tab, TRef) of
-        [{TRef, _SRef}] ->
-            do_apply(MFA);
+    _ = case ets:take(Tab, TRef) of
+        [{TRef}] ->
+            do_apply(MFA, false);
         [] ->
             ok
     end,
```

Comment:
indentation

Reply:
Dto?

Reply:
what do you mean with DTO? (I did google query and still not sure) :-)

a proposal use if you like it

```
handle_info({timeout, TRef, {apply_once, MFA}}, Tab) ->
    _ = case ets:take(Tab, TRef) of
            [{TRef}] ->
                do_apply(MFA, false);                                                                                                                                                                                                                                                                       [] ->
                ok
        end,                                                                                                                                                                                                                                                                                        {noreply, Tab};

```

Reply:
Dto --> dito, meaning something like "likewise" or "the same (as above)". I didn't want to repeat "Seems ok to me, what's wrong with it?" in every answer :wink:

Reply:
Fixed.